src/context/AppContext.tsx

```diff
@@ -61,7 +63,7 @@ export const AppContextProvider = ({ children }: { children: ReactNode }) => {
             GetCurrentUserQueryVariables
           >({
             query: GetCurrentUserDocument,
-            variables: { address },
+            variables: { address: utils.getAddress(address) },
```

Comment:
Gets the checksummed address, which is how we store the user on user creation. Noticed that when creating a user when logged in via metamask, the address provided was all lowercase, and searching in the db is case sensitive.

Reply:
That's what I'm going for. Where would you put it?